pagespeed/apache/header_util.cc

```diff
@@ -115,6 +115,9 @@ void AddResponseHeadersToRequestHelper(const ResponseHeaders& response_headers,
   for (int i = 0, n = response_headers.NumAttributes(); i < n; ++i) {
     const GoogleString& name = response_headers.Name(i);
     const GoogleString& value = response_headers.Value(i);
+    if (strings::StartsWith(name, "@")) {
+      continue;
```

Comment:
curious: does this get hit in tests?  the concerning thing here is whether there's something linking through PSOL into the integrations that we have to catch on each integration.

Reply:
@jmarantz 
I can see that this line should be covered by a test (and perhaps give known implementors with ports similar to mod_pagespeed a head's up, those that do not use ProxyFetch to wire up html rewriting). 

Currently this is not hit in tests, but in the next planned follow-up to this PR, it will be easy to add and end-to-end test because that is where the rubber will start hitting the road:
https://github.com/apache/incubator-pagespeed-mod/pull/1696/files#diff-923e5a0c8d9a28d13112afd3d77b35a0R239